upstream CI (.github/workflows/python-tests.yml) execute these tests
through `python ./bindings/python/tests/test_all.py`, hence they haven't
noticed this so far I assume. However, running the test_all script as
is generates a lot of output so let's fix the check target instead.

--- capstone-5.0.2.orig/bindings/python/Makefile
+++ capstone-5.0.2/bindings/python/Makefile
@@ -59,7 +59,7 @@
 check:
 	@for t in $(TESTS); do \
 		echo Check $$t ... ; \
-		./$$t > /dev/null; \
+		$(CURDIR)/tests/$$t > /dev/null; \
 		if [ $$? -eq 0 ]; then echo OK; else echo FAILED; exit 1; fi \
 	done
 
